apps/opencs/model/tools/referenceablecheck.cpp

```diff
@@ -397,6 +397,41 @@ void CSMTools::ReferenceableCheckStage::containerCheck(
     //checking for name
     if (container.mName.empty())
         messages.push_back (std::make_pair (id, container.mId + " has an empty name"));
+    
+    //checking contained items
+    const std::vector<ESM::ContItem>& itemList = container.mInventory.mList;
+    for (unsigned i = 0; i < itemList.size(); ++i)
```

Comment:
I've looked at the npc and creature data classes, and it looks like they implement the inventory the same way, so I'll make a separate function for comparing inventories. 
I'll go ahead and change the error messages. 
I also didn't realize 'unsigned' was legal until looking it up after seeing it later in the same file. I have no problem with changing it.